actions/views/drafts.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {DispatchFunc} from 'mattermost-redux/types/actions';
+import {setGlobalItem} from 'actions/storage';
+import {PostDraft} from 'types/store/rhs';
+
+export function removeDraft(key: string) {
+    return (dispatch: DispatchFunc) => {
+        localStorage.removeItem(key);
+        dispatch(setGlobalItem(key, {message: '', fileInfos: [], uploadsInProgress: []}));
+    };
+}
+
+export function updateDraft(key: string, value: PostDraft|null) {
```

Comment:
It might not be visible but this returns an action (`setGlobalItem()`), so it has to be dispatched, `dispatch(updateDraft())`.

e.g:

```js
const action = () => setGlobalItem(value)
// the above is the same as doing 
const action = () => dispatch => dispatch(setGlobalItem(value))